packages/paymasters/src/PermitPaymasterUtils.ts

```diff
@@ -80,6 +101,22 @@ export interface PermitInterfaceEIP2612 {
   value: IntString
 }
 
+export interface PaymasterConfig {
+  weth: string
+  tokens: string[]
+  relayHub: string
+  priceFeeds: string[]
+  uniswap: string
+  trustedForwarder: string
+  uniswapPoolFees: number[] | BN[] | string[]
+  gasUsedByPost: number | BN | string
+  permitMethodSignatures: string[]
+  minHubBalance: number | BN | string
+  targetHubBalance: number | BN | string
```

Comment:
Suggestion: make sure that these values can accept `MAX_INT` to signify special configurations:
(MAX_INT does not necessarily be 2^256 here as this will likely break our math... TBD)

`minHubBalance = MAX_INT` - after each transaction perform a swap to ether
`targetHubBalance = MAX_INT` - when replenishing, do not stop and swap all tokens

Reply:
That's already how it works

Reply:
Ok than